renderer/admonition.go

```diff
@@ -0,0 +1,150 @@
+package renderer
+
+import (
+	"fmt"
+
+	"github.com/kovetskiy/mark/parser"
+	"github.com/yuin/goldmark/ast"
+	"github.com/yuin/goldmark/renderer"
+	"github.com/yuin/goldmark/renderer/html"
+	"github.com/yuin/goldmark/util"
+)
+
+// HeadingAttributeFilter defines attribute names which heading elements can have
+var AdmonitionAttributeFilter = html.GlobalAttributeFilter
+
+// A Renderer struct is an implementation of renderer.NodeRenderer that renders
+// nodes as (X)HTML.
+type ConfluenceAdmonitionRenderer struct {
```

Comment:
Please replace Admonition with MkDocsAdmonition. There also is https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/ which have a similar purpose and we likely will have some proper parse for it as some point in time as well.